docs/sources/reference/components/prometheus.scrape.md

```diff
@@ -37,47 +37,56 @@ time), the component reports an error.
-`enable_http2`                | `bool`                  | Whether HTTP2 is supported for requests.                                                               | `true`         | no
-`follow_redirects`            | `bool`                  | Whether redirects returned by the server should be followed.                                           | `true`         | no
-`proxy_url`                   | `string`                | HTTP proxy to send requests through.                                                                   |                | no
-`no_proxy`                    | `string`                | Comma-separated list of IP addresses, CIDR notations, and domain names to exclude from proxying.       |                | no
-`proxy_from_environment`      | `bool`                  | Use the proxy URL indicated by environment variables.                                                  | `false`        | no
-`proxy_connect_header`        | `map(list(secret))`     | Specifies headers to send to proxies during CONNECT requests.                                          |                | no
-
- At most, one of the following can be provided:
+| Name                          | Type                    | Description                                                                                            | Default                                                                   | Required |
+|-------------------------------|-------------------------|--------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|----------|
+| `targets`                     | `list(map(string))`     | List of targets to scrape.                                                                             |                                                                           | yes      |
+| `forward_to`                  | `list(MetricsReceiver)` | List of receivers to send scraped metrics to.                                                          |                                                                           | yes      |
+| `job_name`                    | `string`                | The value to use for the job label if not already set.                                                 | component name                                                            | no       |
+| `extra_metrics`               | `bool`                  | Whether extra metrics should be generated for scrape targets.                                          | `false`                                                                   | no       |
+| `enable_protobuf_negotiation` | `bool`                  | Deprecated: use `scrape_protocols` instead.                                                            | `false`                                                                   | no       |
```

Comment:
Under the table, we should clarify that folks who have `enable_protobuf_negotiation = true` should switch to `scrape_protocols = ["PrometheusProto", "OpenMetricsText1.0.0", "OpenMetricsText0.0.1", "PrometheusText0.0.4"]`